docs/lms-service-api-document.md

```
@@ -1883,10 +1889,109 @@ organisationid: <organisation-id>
 ```
 
+#### Clone Module
+
+**Endpoint**: `POST /modules/{moduleId}/clone`
+
+**HTTP Method**: POST
+
+**Description**: Clone a module with all its lessons and submodules
+
+**Headers**:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)